examples/api-gateway-websocket-chat-app/variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "aws_access_key" {
+  description = ""
+}
+
+variable "aws_secret_key" {
+  description = ""
+}
+
+variable "aws_region" {
+  description = ""
```

Comment:
Can you add a default value for `aws_region`? This should be as out-of-the-box as possible

Reply:
I'll default it to `us-west-2`.